deployment/requirements.txt

```diff
@@ -4,7 +4,6 @@ fabtools==0.19.0
 paramiko==1.14.0
 pycrypto==2.6.1
 cfn-pyplates==0.5.0
-awscli==1.10.5
+awscli==1.14.26
 aws-shell==0.1.0
-boto3==1.4.7
```

Comment:
boto3 is required by stack.py and even if it is a dependency of another package, it should be specified explicitly with an exact version to avoid potential bugs caused by silent upgrades (https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#meta)